rootfs/etc/__init__.py

```diff
@@ -0,0 +1,2 @@
+import pkgutil
```

Comment:
Sorry why is this called rootfs/etc?  It seems like it's the celery parts, and you make this object with this line:

app_celery.config_from_object('etc.celery.celery_config')

on imageworker.py:44 

Etc is a bad name.  I was confused when I saw it before, and I'm even more confused now.  Are you actually copying these files to /etc?  That seems really not good, if you can at all avoid it.  Why not just put it in the same module with the rest?

Reply:
The idea for this /etc, actually under /app within the imgserv container, is to place all the imgserv and related service configurations. I can easily see these 'external' settings mapped to host paths, for the ease of modification during deployment or integration testing.

Reply:
I think config would be a better name than etc.  I don't think these paths will ever be different, since it will only run in the container?

Reply:
In Linux, /etc contains configuration files.